src/lib/common/include/sol-platform.h

```diff
@@ -73,9 +73,10 @@ extern "C" {
 const char *sol_platform_get_board_name(void);
 
 /**
- * @brief Retrieves, in @a id, the machine-id present in the file system.
+ * @brief Retrieves the machine-id present in the file system as a string.
  *
```

Comment:
This patch is in this pull req because this function will be used by OIC implementation, but it is not directly related to OIC.
Sol-platform has functions related to soletta platform. Machine-ID is an UUID that defines a machine, i.e. a device. This is used by OIC as the device id, but it is used in other Soletta modules too.